vector/src/main/java/im/vector/app/EmojiCompatWrapper.kt

```diff
@@ -23,8 +23,12 @@ import timber.log.Timber
 import javax.inject.Inject
 import javax.inject.Singleton
 
+fun interface EmojiSpanify {
```

Comment:
extracted out the processing to allow the tests to provide their own implementation that doesn't rely on any singleton flags

Reply:
`fun interface` 🎉